packages/dd-trace/test/setup/mocha.js

```diff
@@ -17,6 +17,9 @@ global.withVersions = withVersions
 global.withExports = withExports
 global.withNamingSchema = withNamingSchema
 global.withPeerService = withPeerService
+global.testAgentServiceName = null
```

Comment:
The existing global stateless test helper functions are used across a wide number of tests.

It looks like these new globals are very stateful and were added only for a single test file, `packages/dd-trace/test/plugins/agent.js`? It also looks like a pretty tight coupling now between mocha.js and agent.js.

A better way to access and mutate these values would be to stick them into another JavaScript module via exports. This makes them singleton values but no longer with the side effects of being globals. That said, I think there would still be a tighter-than-needed coupling between mocha.js and agent.js.

Should we hop in a meeting to figure out the best course of action?